lib/libzrepl/data_conn.c

```diff
@@ -1430,10 +1430,11 @@ uzfs_zvol_rebuild_scanner(void *arg)
 			    == 1)
 				sleep(5);
 #endif
+			uint64_t cio_seq = hdr.checkpointed_io_seq;
 snap_reverify:
 			if (snap_zv == NULL) {
 				rc = uzfs_get_snap_zv_ionum(zinfo,
-				    hdr.checkpointed_io_seq, &snap_zv);
+				    cio_seq, &snap_zv);
```

Comment:
:+1:

Reply:
actually, I was thinking of how to test these things. `checkpointed_io_seq` is the thing which our rebuild depends on.